db/migrate/20241002052203_create_mentions.rb

```diff
@@ -0,0 +1,13 @@
+class CreateMentions < ActiveRecord::Migration[7.0]
+  def change
+    create_table :mentions do |t|
+      t.integer :mention_id, :mention_id, false
+      t.integer :mentioned_id :mentioned_id, false
```

Comment:
手元で実行しようとしたのですが、syntax errorになってますね。

```suggestion
      t.integer :mention_id, null: false
      t.integer :mentioned_id, null: false
```

ではないですか？

Reply:
そうでした😵‍💫　おそらく記憶に頼って書いてしまったんだと思います。今回のように間違って覚えているパターンもあると思うので、その都度確認して書くようにします。